plugins/http/Cargo.toml

```diff
@@ -35,14 +35,14 @@ url = { workspace = true }
 data-url = "0.3"
 
 [features]
+default = ["rustls-tls", "http2", "charset", "macos-system-configuration"]
 multipart = ["reqwest/multipart"]
 json = ["reqwest/json"]
 stream = ["reqwest/stream"]
 native-tls = ["reqwest/native-tls"]
 native-tls-vendored = ["reqwest/native-tls-vendored"]
-rustls-tls = ["reqwest/rustls-tls"]
-default-tls = ["reqwest/default-tls"]
```

Comment:
we need to keep these two features, so users can disable default features and customize as they want

Reply:
Whoops, rustls was removed by accident, will revert that tomorrow.
default-tls was removed because it's an alias for native-tls (or the other way around) and I think the name is pretty misleading (probably not the right word for it idk) or not self explanatory at least.

Reply:
Oh wait, just noticed that I just re-ordered it. The rustls flag is still there just further down below lol

Reply:
Oh and another reason why I removed default-tls is because I used rustls as the default and thought that may also be confusing when default-tls is not default

Reply:
cool